steelthread-0/walkthrough.md

```diff
@@ -97,41 +106,296 @@ Restart the booster services
 
 ```
 oc rollout latest nodejs-messaging-work-queue-frontend
-oc rollout latest nodejs-messaging-work-queue-worker
 ```
 
-# Setup HTTP connection to the messaging booster in Fuse Ignite
-- Open the Fuse ignite console and open the connections again. 
+In the log of the application you should see something like:
+
+```
+frontend-nodejs-aad7: Connected to AMQP messaging service at messaging.enmasse-eval.svc:5672
+```
+
+This is how we know it has connected to the AMQP service succesfully. 
+
+# Setup Enmasse connection in Fuse Ignite
+
+Back in your ```evals``` project, You will see a binding and a binding secret as part of the Enmaase provisioned service. There is an option to view this secret. Click on view and click on reveal. Take note of the following values
+
+- messagingHost
+- username
+- password
+
+- Open the Fuse Ignite route 
+- Login in to the Fuse Ignite console using the evals@example.com user 
+- Select connections and create connection
+- Find the AMQP connection and select it. 
+- Fill in the Connection URI Field with the a URI that is in the following format  ```amqp://<messagingHost>:5672?amqp.saslMechanisms=PLAIN```
```

Comment:
Can probably reference ```messaging.enmasse-eval.svc``` rather than ```<messagingHost>``` as we mention ```messaging.enmasse-eval.svc``` elsewhere in the guide

Reply:
Will use the expected host `messaging.enmasse-eval.svc`